.gitignore

```diff
@@ -61,3 +61,9 @@ lib/qtscript-bytearray/*.cc
 .idea
 *.vscode
 compile_commands.json
+
+# Developers might use `--settingsPath ./devsettings/`
+devsettings
+
+# Nix build result
+result
```

Comment:
I'm undecided on this one, because it is a very generic name. The pattern should also be more specific, i.e. use `/result` to avoid exclusions in subdirectories.

Reply:
Done.